### PR TITLE
fix typo and add note on sockcm error

### PR DESCRIPTION
**Pull request by quasiben**
cc @pentschev

### CODE REVIEW

File: docs/source/ucx-debug.rst [hunk 1]
```diff
@@ -118,3 +119,8 @@ Experimental Debugging
 A list of problems we have run into along the way while trying to understand performance issues with UCX/UCX-Py:
 
 - System-wide settings environment variables. For example, we saw a system with ``UCX_MEM_MMAP_HOOK_MODE`` set to ``none``.  Unsetting this env var resolved problems: https://github.com/rapidsai/ucx-py/issues/616 .  One can quickly check system wide variables with ``env|grep ^UCX_``.
+
+
+- ``sockcm_iface.c:257 Fatal: sockcm_listener: unable to create handler for new connection``.  This is an error we've seen when limits are place on the number
+of file descriptors.  User have two choices for resolving this issue: increase the ``open files`` limit (check ulimit configuration) or use ``RDMACM`` when establishing
+a connection ``UCX_SOCKADDR_TLS_PRIORITY=rdmacm``
```
Comment (pentschev) [inline docs/source/ucx-debug.rst:126]: We have to be clear here that `RDMACM` is available only with InfiniBand.

File: docs/source/ucx-debug.rst [hunk 2]
```diff
@@ -18,7 +18,8 @@ System Configuration
     mlx5_2 port 1 ==> ib2 (Up)
     mlx5_3 port 1 ==> ib3 (Up)
 
-``ucx_info -d`` and ``ucx_info -p -u t`` are helpful commands to display what UCX understands about the underlying hardware
+``ucx_info -d`` and ``ucx_info -p -u t`` are helpful commands to display what UCX understands about the underlying hardware.
+For example, we can check
```
Comment (pentschev) [inline docs/source/ucx-debug.rst:22]: Something seems to be missing here, could you check this @quasiben ?

File: docs/source/ucx-debug.rst (hunk 1 above)
Comment (quasiben) [reply docs/source/ucx-debug.rst:126]: Good point

File: docs/source/ucx-debug.rst (hunk 2 above)
Comment (quasiben) [reply docs/source/ucx-debug.rst]: yup, left off my examples.  included them in the last commit

File: docs/source/ucx-debug.rst [hunk 3]
```diff
@@ -118,3 +134,9 @@ Experimental Debugging
 A list of problems we have run into along the way while trying to understand performance issues with UCX/UCX-Py:
 
 - System-wide settings environment variables. For example, we saw a system with ``UCX_MEM_MMAP_HOOK_MODE`` set to ``none``.  Unsetting this env var resolved problems: https://github.com/rapidsai/ucx-py/issues/616 .  One can quickly check system wide variables with ``env|grep ^UCX_``.
+
+
+- ``sockcm_iface.c:257 Fatal: sockcm_listener: unable to create handler for new connection``.  This is an error we've seen when limits are place on the number
+of file descriptors and occurs when ``SOCKCM`` is used for establishing connections.  User have two choices for resolving this issue: increase the
+``open files`` limit (check ulimit configuration) or use ``RDMACM`` when establishing a connection ``UCX_SOCKADDR_TLS_PRIORITY=rdmacm``.  ``RDMACM``
+is only available also using InfiniBand devices.
```
Comment (pentschev) [inline docs/source/ucx-debug.rst:142]:
```suggestion
is only available using InfiniBand devices.
```

**Review by pentschev — CHANGES_REQUESTED** [review]

**Comment by pentschev** [comment]
One minor change request, apart from that everything else looks good. Thanks @quasiben !

**Comment by quasiben** [comment]
Thanks @pentschev

**Review by pentschev — APPROVED** [review]

**Pull request merged**